demosplan/DemosPlanCoreBundle/Logic/Factory/PercentageDistributionFactory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+
+namespace demosplan\DemosPlanCoreBundle\Logic\Factory;
+
+
+use DemosEurope\DemosplanAddon\Contracts\Factory\PercentageDistributionFactoryInterface;
+use DemosEurope\DemosplanAddon\Contracts\ValueObject\PercentageDistributionInterface;
+use demosplan\DemosPlanCoreBundle\ValueObject\PercentageDistribution;
+
+class PercentageDistributionFactory implements PercentageDistributionFactoryInterface
+{
+    public function createPercentageDistribution(int $total, array $absolutes): PercentageDistributionInterface
+    {
+        return new PercentageDistribution($total, $absolutes);
+    }
+}
```

Comment:
Just as an aside, this doesn't fit into the PR description and should have probably been it's own PR if we want to aim for change topic separation.

Reply:
Is moved to https://github.com/demos-europe/demosplan-core/pull/1002